.vscode/extensions.json

```diff
@@ -1,9 +1,10 @@
 {
   "recommendations": [
     "Vue.volar",
-    "Vue.vscode-typescript-vue-plugin",
+    "Vue.vscode-typescript-vue-plugin", // deprecated
```

Comment:
:+1: This was deprecated fairly recently. We'll remove the line once Vue init stops putting it here.